example/local/example1.py

```diff
@@ -165,7 +165,7 @@ async def on_message(message):
 
         await vc.shuffle()
```

Comment:
I mistaked.

vc.shuffle is not coroutine.

```suggestion
        vc.shuffle()
```